docs/source/APIs/index.rst

```diff
@@ -1,7 +1,9 @@
-API Reference
-===============
+API References
+=================
 
 .. toctree::
-   :maxdepth: 1
+   :maxdepth: 2
 
-   functions.rst
+   functions/functions.rst
+   data/data.rst
+   transforms/transforms.rst
```

Comment:
Minor: I would order this as data structures, then transforms, then functions. I think it will be easier to understand the functions once you look at the other two categories.

Reply:
Updated